devices/tuya.js

```diff
@@ -720,6 +720,18 @@ const fzLocal = {
             return result;
         },
     },
+    gateway_connection_status: {
+        cluster: 'manuSpecificTuya',
+        type: ['commandMcuGatewayConnectionStatus'],
+        convert: async (model, msg, publish, options, meta) => {
+            // "payload" can have the following values:
```

Comment:
Not sure about how @Koenkk handles comments in the code in this library.
For my part, I never have commented out lines in the code itself as a best practice. 

These lines here should belong in a documentation IMHO - @Koenkk I did not find a documentation covering the gateway connection status. Should be somewhere close to [this](https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html) I think .

I can try to sum up my learnings from this PR and from the discussion in issue #15729 and create a PR to add some info about that in the documentation, if that is ok for you.

Reply:
These are valid comments IMO, they are not code, just explains what it does. 

These lines are taken from the TuYa developer documentation.